app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,32 @@
+class VotesController < ApplicationController
```

Comment:
You're creating a unique style for yourself using `attr_reader` and `custom_initializer`